codeship-services.yml

```diff
@@ -34,7 +34,7 @@ e2e:
   encrypted_env_file: ./ci-env-vars.encrypted
   cached: true
   encrypted_dockercfg_path: dockercfg.encrypted
-  default_cache_branch: 'dev'
+  default_cache_branch: 'master'
```

Comment:
Will we run these steps on`master`? I don't think so right?

Reply:
Oh yeap you are right... I assumed that CI/CD will pick up the commit identity between branches, but I guess this is not the case